server/controllers/article.controller.js

```diff
@@ -40,5 +40,31 @@ class ArticleController {
     }
     return Response(res, 201, successMessage, { article });
   }
+
+  static async getAllArticles(req, res) {
+    /**
+     * @description - Controller to get all articles endpoint
+     *
+     * @static
+     * @param {object} request
+     * @param {object} response
+     * @returns {object}
+     * @memberof getAllArticles
+    */
+    const successMessage = 'Articles successfully retrieved';
```

Comment:
@BukkyOmo There is no need for this variable here. You are using this once, hence kindly use it where you are supposed to use it. 
Assigning a variable consumes more memory space.